README.md

```diff
@@ -6,14 +6,26 @@ applications within the GOV.UK stack.
 These are used to verify releases and also to provide Icinga alerts for major
 features.
 
+The tests are run against integration, staging and production environments, and are triggered by deployments of most GOV.UK applications, CDNs and associated dependencies (`grep -i "smokey" modules/govuk_jenkins/templates/jobs/*` in the [govuk-puppet repository](https://github.com/alphagov/govuk-puppet) for a full list).
+
 ## Technical documentation
 
 The smoke tests are based on [Cucumber](https://cucumber.io/). We use feature
 files to describe single applications (eg
 [`whitehall`](https://github.com/alphagov/whitehall),
 [`frontend`](https://github.com/alphagov/frontend)) or [cross-application behaviour](features/gov_uk.feature).
 
-### Running the tests
+### Installation
+
+Smokey requires Java to be installed, because of its [use of the BrowserMob Proxy](#use-of-browsermob-proxy). [Download Java from the Oracle website](https://www.oracle.com/technetwork/java/javase/downloads/index.html) (you'll need to create an Oracle account). Note that if you're using the Development VM then Java is already installed.
```

Comment:
Can we use OpenJDK instead of Oracle's version? That can be installed using `brew cask install java` without the need for an Oracle account.

Reply:
Wish I'd known that! 😄 Will update 👍